src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -1495,6 +1496,14 @@ namespace ngraph
                     output_index++;
                 }
             }
+            template <>
+            void CPU_Emitter::EMITTER_DECL(ngraph::op::StopGradient)
+            {
+                writer.block_begin();
+                writer << "memcpy(" << out[0].get_name() << ", " << args[0].get_name() << ", "
+                       << out[0].get_size() * out[0].get_element_type().size() << ");\n";
+                writer.block_end();
+            }
```

Comment:
I think CPU has a no-op emitter you can use instead to avoid the memcpy?

Reply:
nop does not seem to do pass through inputs (stop gradient expects identity/copy function on forward)

Reply:
Why not just pattern-match this into oblivion during passes shared by all transformers? Then you don't need anything for the transformers.